prometheus/converters.go

```diff
@@ -93,6 +94,8 @@ func ReadRequestToInfluxQLQuery(req *remote.ReadRequest, db, rp string) (*influx
 // condFromMatcher converts a Prometheus LabelMatcher into an equivalent InfluxQL BinaryExpr
 func condFromMatcher(m *remote.LabelMatcher) (*influxql.BinaryExpr, error) {
 	var op influxql.Token
+	var RHS influxql.Expr
```

Comment:
Can `RHS` be lowercase? Since it's a local variable, I think that's the standard for local variables.